pkg/telegram-bot/telegram-bot.go

```diff
@@ -1 +1,421 @@
 package telegrambot
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/joho/godotenv"
+	"gopkg.in/telebot.v4"
+)
+
+const (
+	USER       = "User"
+	ADMIN      = "Admin"
+	SUPERADMIN = "SuperAdmin"
+)
+
+type user struct {
```

Comment:
it is better to define user model and its receiver functions in model directory and use it in this file